config/initializers/cookies_serializer.rb

```diff
@@ -2,4 +2,4 @@
 
 # Specify a serializer for the signed and encrypted cookie jars.
 # Valid options are :json, :marshal, and :hybrid.
-Rails.application.config.action_dispatch.cookies_serializer = :hybrid
+Rails.application.config.action_dispatch.cookies_serializer = :json
```

Comment:
Will this log everyone out?

Reply:
I don't think so – afaik hybrid deserializes either, json only deserializes a particular one. I will deploy to staging and check <3

Reply:
Testing it, it didn't log me out, so no, I expect it won't.